spec/features/person_maintenance_spec.rb

```diff
@@ -180,6 +191,14 @@
       end
     end
 
+    context "with an external user", user: :external_user do
+      it "action is not allowed without login" do
```

Comment:
Maybe change the test name to reflect that the action is not allowed for an external user